airgun/session.py

```diff
@@ -214,8 +216,7 @@ def __exit__(self, exc_type, exc_value, traceback):
         if self.browser is None:
             # browser was never started, don't do anything
             return
-        LOGGER.info(
-            u'Stopping UI session %r for user %r', self.name, self._user)
+        LOGGER.info('Stopping UI session %r for user %r', self.name, self._user)
```

Comment:
f-string?

Reply:
Not for logger calls, the logger does its own formatting, and takes the fields as arguments directly.

Reply:
python's base logger really needs an update..